src/main/java/com/conveyal/datatools/manager/extensions/mtc/MtcFeedResource.java

```diff
@@ -88,49 +100,45 @@ public void importFeedsForProject(Project project, String authHeader) throws IOE
             in.close();
 
             String json = response.toString();
-            RtdCarrier[] results = mapper.readValue(json, RtdCarrier[].class);
-            for (int i = 0; i < results.length; i++) {
-                //                    String className = "RtdCarrier";
-                //                    Object car = Class.forName(className).newInstance();
-                RtdCarrier car = results[i];
-                //LOG.info("car id=" + car.AgencyId + " name=" + car.AgencyName);
-
+            RtdCarrier[] carriers = mapper.readValue(json, RtdCarrier[].class);
```

Comment:
It's possible for an ObjectMapper instance to parse directly from an InputStream thus avoiding the need to read to a String object.